extras/ams_wrapper/start_ams.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Adding `-e` flag here would be nice, so script will exit immediately in case of error